app/models/merchant.rb

```diff
@@ -5,7 +5,7 @@ class Merchant < ApplicationRecord
   has_many :customers, through: :invoices
   has_many :transactions, through: :invoices
 
-  enum status: {"enable": 0, "disable": 1}
+  enum status: { "enable": 0, "disable": 1 }
```

Comment:
I had to create an enum for the item model as well, and I may have not spaced it correctly haha